lib/interfaces/bdd.js

```diff
@@ -91,6 +91,20 @@ module.exports = function(suite){
     };
 
     /**
+     * Do not randomize.
+     */
+    context.describe.noRandom = function func(title, fn) {
```

Comment:
See the last hunks of this PR. It's necessary to add these markers, even in mocha test suite itself.